keps/sig-node/2008-forensic-container-checkpointing/README.md

```diff
@@ -292,14 +329,94 @@ Checkpointing containers will be possible again.
 
 ###### Are there any tests for feature enablement/disablement?
 
-Currently no.
+Currently the test will automatically be skipped if the feature is not enabled.
+
+### Rollout, Upgrade and Rollback Planning
+
+Does not apply as the feature is an additional API endpoint with no
```

Comment:
Some general confusion on my part (the kep here is very minimal), why would a forensic API like this be triggerable remotely by  using a kubelet API versus being specific to the CRI on the host?

Reply:
I don't understand this question. Sorry.

Reply:
The idea was to start with a kubelet API for this limited forensic use case and then tackle kubernetes API separately in a different KEP.